x-pack/plugins/endpoint/server/services/endpoint/endpoint_query_builders.test.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { httpServerMock, loggingServiceMock } from '../../../../../../src/core/server/mocks';
+import { EndpointConfigSchema } from '../../config';
+import { AllEndpointsQueryBuilder } from './endpoint_query_builders';
+
+describe('test query builder', () => {
+  describe('test query builder request processing', () => {
+    it('test query with no filter defaults', async () => {
+      const mockRequest = httpServerMock.createKibanaRequest({
+        body: {},
+      });
+      const searchQueryBuilder = new AllEndpointsQueryBuilder(mockRequest, {
+        logFactory: loggingServiceMock.create(),
+        config: () => Promise.resolve(EndpointConfigSchema.validate({})),
+      });
+      const query = await searchQueryBuilder.toQueryParams();
+      expect(query).toEqual({
```

Comment:
Would you explain the purpose of this test?

Reply:
update the test describe

Reply:
Sorry for the confusion. Could you explain what you hope to accomplish by having this test? It seems to just check the return value of a function: https://github.com/elastic/kibana/pull/53861/files/6327a4fbdde17b600f8d68d0e721e9f676f6f066#diff-0ab9ae9ab2c5fdf664d3779d6058b9a1R20

Reply:
To get the collapsed list of endpoint, we have a particular query that has to be built. We are testing that the query is created.